common/main.go

```diff
@@ -0,0 +1,66 @@
+package common
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Log contains the default logger to use.
+var Log = logrus.WithFields(logrus.Fields{
+	"service": "app-exposer",
+	"art-id":  "app-exposer",
+	"group":   "org.cyverse",
+})
```

Comment:
This bit of code was duplicated in a few places, so I thought I'd put it in the `common` package to eliminate some of the repetition.